doc/refman/RefMan-syn.tex

```diff
@@ -1322,6 +1322,57 @@ \subsubsection{\tt Print Scopes\comindex{Print Scopes}}
 %except that abbreviations are used for printing (unless the modifier
 %\verb=(only parsing)= is given) while syntactic definitions were not.
 
+\section{Numeral Notations
+\comindex{Numeral Notation}}
+
+Starting with version 8.8, the following command allows to customize
```

Comment:
This is not proper English, though it seems common among Coq commit messages.  Use "allows customization of" or "allows the user to customize", not "allows to customize".

Reply:
Done, thanks